graphz/src/main/scala/coop/rchain/graphz/Graphz.scala

```diff
@@ -2,22 +2,21 @@ package coop.rchain.graphz
 
 import java.io.FileOutputStream
 
-import cats._, cats.data._, cats.implicits._
+import cats._
 import cats.effect.Sync
+import cats.implicits._
 import cats.mtl._
 
 trait GraphSerializer[F[_]] {
   def push(str: String, suffix: String = "\n"): F[Unit]
 }
 
-class StringSerializer[F[_]: Applicative: MonadState[?[_], StringBuffer]]
-    extends GraphSerializer[F] {
+class StringSerializer[F[_]: MonadState[?[_], StringBuffer]] extends GraphSerializer[F] {
   def push(str: String, suffix: String): F[Unit] =
     MonadState[F, StringBuffer].modify(sb => sb.append(str + suffix))
 }
 
-class ListSerializer[F[_]: Applicative: MonadState[?[_], Vector[String]]]
-    extends GraphSerializer[F] {
+class ListSerializer[F[_]: MonadState[?[_], Vector[String]]] extends GraphSerializer[F] {
```

Comment:
Why do we actually need it here? :thinking: And why does a serializer return `Unit`? :thinking: